deepcell_tracking/tracking_test.py

```diff
@@ -71,10 +71,11 @@ def _get_dummy_tracking_data(length=128, frames=3,
 class DummyModel(object):  # pylint: disable=useless-object-inheritance
 
     def predict(self, data):
-        if isinstance(data, list):
-            batches = 0 if not data else len(data[0])
+        if isinstance(data, dict):
+            # Grab a random value from the data dict and select batch dim
+            batches = 0 if not data else next(iter(data.values())).shape[0]
         else:
-            batches = len(data)
+            batches = next(iter(data.values())).shape[0]
```

Comment:
If the data is not a `dict` what happens here?

Reply:
It will fail. That `else` block is not structured correctly.